finality-aleph/src/network/substrate.rs

```diff
@@ -27,6 +28,18 @@ pub struct ProtocolNetwork {
     last_status_report: time::Instant,
 }
 
+impl Borrow<Box<dyn sc_network::config::NotificationService>> for ProtocolNetwork {
+    fn borrow(&self) -> &Box<dyn sc_network::config::NotificationService> {
+        &self.service
+    }
+}
+
+impl BorrowMut<Box<dyn sc_network::config::NotificationService>> for ProtocolNetwork {
```

Comment:
How about creating a type alias for `Box<dyn sc_network::config::NotificationService>` ?

Reply:
added